tests/integration/test_remoted/conftest.py

```diff
@@ -23,3 +25,26 @@ def restart_remoted(get_configuration, request):
         control_service('start', daemon=DAEMON_NAME)
     except sb.CalledProcessError:
         pass
+
+
+@pytest.fixture(scope="module")
+def create_agent_group():
+    """Temporary creates a new agent group for testing purpose, must be run only on Managers."""
+
+    subprocess.run(["/var/ossec/bin/agent_groups", "-q", "-a", "-g", "testing_group"])
+
+    with open("/var/ossec/etc/shared/testing_group/agent.conf", "w") as f:
+        f.write('''
+                <!-- custom configuration file
+                    example for agents! -->
+                <agent_config>
+                    <localfile>
+                        <location>/var/log/my_test.log</location>
+                        <log_format>syslog</log_format>
+                    </localfile>
+                </agent_config>
+                ''')
+
+    yield
+
+    subprocess.run(["/var/ossec/bin/agent_groups", "-q", "-r", "-g", "testing"])
```

Comment:
The group name was called `testing_group`, right?

Reply:
Oh, you're right! I've fixed it here, thanks!

https://github.com/wazuh/wazuh-qa/pull/1115/commits/42a1c0e6891c246afc6a0292984a46fd47d5a26f